examples/webhooks/webhooks.go

```diff
@@ -0,0 +1,64 @@
+/*
+Package main - send_op_return example
+*/
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	walletclient "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/notifications"
+	"github.com/bitcoin-sv/spv-wallet/models"
+)
+
+func main() {
+	defer examples.HandlePanic()
```

Comment:
Wow, great idea with this example 😳
What do you think about handling signals, something like this:
```go
sigChan := make(chan os.Signal, 1)
signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM)
go func() {
    <-sigChan
    if err := server.Shutdown(context.Background()); err != nil {
        panic(err)
    }
    if err := wh.Unsubscribe(context.Background()); err != nil {
        panic(err)
    }
}()
```
it should make sure that if any signal with code 1 appears it will shut down services "gracefully".